wiki/Beatmaps/en.md

```diff
@@ -3,57 +3,153 @@
+
+## Downloading beatmaps
+
+Beatmaps can be obtained through a few different ways:
+
+### Beatmap listing
+
+Beatmaps from various categories are available from the [beatmap listing](https://osu.ppy.sh/beatmapsets) on the website. They can be filtered according to certain conditions (e.g. beatmaps from [osu!taiko](/wiki/Game_mode/osu!) created by a certain mapper), and then downloaded one by one. [osu!supporters](/wiki/osu!supporter) have access to an extended set of filters, such as an ability to see beatmaps on which they achieved a certain rank.
+
+### osu!direct
+
+osu!direct is the beatmap listing built into the game client, which is used for quick access and provides a set of features similar to the listing on the website. It is only available to the [osu!supporters](/wiki/osu!supporter).
+
+### Beatmap packs
+
+*Main article: [Beatmap Packs](Packs)*
+
+Beatmapsets that were ranked in the same month, or share the same theme (e.g. the same artist), are packed for bulk download into archives, which are called the beatmap packs. The list of all official packs is available on the [website](https://osu.ppy.sh/beatmaps/packs).
+
+### Unofficial sources
+
+There are also secondary sources of beatmaps, which are **neither checked nor operated by the osu!team**. However, they are available on the forums and appreciated by the community. Such sources include:
+
+- third-party beatmap mirrors hosted off a platform;
+- player-made packs and compilations shared via [p2p-networks](https://en.wikipedia.org/wiki/Peer-to-peer "Wikipedia"), such as BitTorrent.
+
+## Beatmaps and community
+
+### Leaderboards
+
+osu! players across all game modes use beatmaps to [rank up globally](/wiki/Performance_points) and [compete against each other](/wiki/Ranking). osu! has different types of beatmap-specific leaderboards, all of which, except the first one, are only available to [osu!supporters](/wiki/osu!supporter):
+
+- Global ranking, which works across the active playerbase;
+- Global ranking for every combination of [game modifiers](/wiki/Game_modifier);
+- Country-specific ranking made of players bearing the same flag;
+- Friend ranking that shows a player's position on the map compared to their friends.
+
+### Charts and Spotlights
+
+<!-- TODO(TicClick): charts, as well as Chart Assembly Team, need to be referenced here when they receive a dedicated article (issue #4685) -->
+
+<!-- TODO(TicClick): would be very cool to have a separate article for osu!lazer as well (issue #4686) -->
+
+*Main article: [Beatmap Spotlights](/wiki/Beatmap_Spotlights)*
+
+Since its early days, the community has had various ways to showcase unique and excellent maps. One of the first documented approaches was monthly and seasonal charts consisting of a small subset of ranked beatmaps, where every month's top players from leaderboards displaying total score across these maps would be awarded with [osu!supporter](/wiki/osu!supporter).
```

Comment:
```suggestion
Since its early days, the community has had various ways to showcase unique and excellent maps. One of the first documented approaches were monthly and seasonal charts consisting of a small subset of ranked beatmaps, where top players from leaderboards displaying total score across these maps would be awarded with [osu!supporter](/wiki/osu!supporter).
```